enhancements/cluster-logging/cluster-logging-v2-apis.md

```diff
@@ -0,0 +1,515 @@
+
+* managing and deploying observability operands (e.g. LokiStack, ClusterLogForwarder, Tracing) and console plugins (e.g log-view-plugin)
+
+**loki-operator** is an operator responsible for managing a loki stack
+
+**cluster-logging-operator** is an operator responsible for managing log collection and forwarding
+
+The cluster administrator does the following:
+
+1. Deploys the Red Hat **cluster-observability-operator**
+1. Deploys the Red Hat **loki-operator**
+1. Deploys an instance of **LokiStack** in the `openshift-logging` namespace
+1. Deploys the Red Hat **cluster-logging-operator**
+1. Creates a **ClusterLogForwarder** custom resource for the **LokiStack**
+
+The **cluster-observability-operator**:
+1. Deploys the logging-view-plugin for reading logs in the OpenShift console
+
+The **loki-operator**:
+1. Deploys the **LokiStack** for storing logs on-cluster
+
+The **cluster-logging-operator**:
+
+1. Deploys the log collector to forward logs to log storage in the `openshift-logging` namespace
+
+### API Extensions
+
+This API defines the following opinionated input sources which is a continuation of prior cluster logging versions:
+
+* **application**: Logs of container workloads running in all namespaces except **default**, **openshift***, and **kube***
+* **infrastructure**: journald logs from OpenShift nodes and container workloads running only in namespaces **default**, **openshift***, and **kube***
+* **audit**: The logs from OpenShift nodes written to the node filesystem by: Kubernetes API server, OpenShift API server, Auditd, and OpenShift Virtual Network (OVN).
+
+These are **well-known** input sources that can be referenced by a pipeline without an explicit input specification.
+
+More explicit specification of **audit** and **infrastructure** logs is allowed by creating a named input of that type and specifiying at least one of the allowed sources.
+
+#### CluserLogForwarer CustomResourceDefinition:
+
+Following defines the next version of a ClusterLogForwarder. **Note:** The next version of this resources is part of a new API group to align log collection with
+the objectives of Red Hat observability.
+
+```yaml
+    apiVersion: "observability.openshift.io/v1"
+    kind: ClusterLogForwarder
+    metadata:
+      name:
+    spec:
+      serviceAccount:
+        name:
+      collector:
+        resources:       #corev1.ResourceRequirements
+          limits:        #cpu, memory
+          requests:
+        nodeSelector:    #map[string]string
+        tolerations:     #corev1.Toleration
+      inputs:
+      - name:
+        type:                      #enum: application,infrastructure,audit
+        application:
+          selector:                #labelselector
+          includes:
+          - namespace:
+            container:
+          excludes:
+          - namespace:
+            container:
+          tuning:
+            ratelimitByContainer:  # map[string]RateLimit, glob (e.g ngnix: {recordsPerSecond: 20})
```

Comment:
Since the application selector already has includes/excludes for container names, I think we can simplify and just have a single rate limit for all the selected containers:

```suggestion
            ratelimitPerContainer:  # rate limit applied to each container selected by this input
                  recordsPerSecond:  #int  (no multiplier, a each container only runs on one node at a time.)
```